mipsevm/evm.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/bash -e
 (cd ../ && npx hardhat compile > /dev/null)
-go build && ./mipsevm $@
+go build && ./mipsevm/mipsevm $@
```

Comment:
That's only OK if you call `evm.sh` from the root, but in the `README`, we call it is as `(cd mipsevm && ./evm.sh)` which doesn't work with this change.

But on the other hand, this assume it is run from the root of the repo, or it doesn't find `mipigo/minigeth.bin`.

Should be changed to `(go build && cd .. && ./mipsevm/mipsevm $@)` for now, while we figure out how to better structure all these scripts.